### DISCUSSION TITLE
Best Framework & libraries in Golang for Production based applications?

### DISCUSSION THREAD

**Discussion by sumit1git** [slack-go/slack#1141] [Q&A] [answered]
I have to develop the Golang application production environment we need to know the below points
1. Best Framework for microservices
2. Best LIbrary for Unit Testing 
3. Library for security with approaches
4. Best way to use gRPC with Golang

Please help ASAP

**Comment by kanata2** [comment, marked as answer]
This is not a place for general Go discussions but for the Slack SDK, so please ask your questions elsewhere, for example on Reddit or SlackOverflow.
